src/components/Card/Card.jsx

```diff
@@ -70,43 +66,41 @@ class Card extends React.Component {
 
     return (
       <div
-        ref={(c) => { this.outer = c; }}
+        ref={c => {
+          this.outer = c;
+        }}
         className={cardStyle}
         style={{ minWidth }}
         onClick={onClick}
         {...getDataAttrs(data)}
       >
         <div className={style.inner}>
-          { (title || actions) &&
+          {(title || actions) && (
             <div className={style.top}>
-              { title &&
-                <Header h3>
-                  { title }
-                </Header>
-              }
-              { actions &&
-                <div className={style.cardActions}>
-                  { actions.map((action) => action)}
-                </div>
-              }
+              {title && <Header h4>{title}</Header>}
```

Comment:
wouldn't this affect all existing components using `<Card />`? are we sure we wanna set all headers to `h4`?